content/docs/strict-mode.md

```diff
@@ -127,3 +128,65 @@ The legacy context API is error-prone, and will be removed in a future major ver
 ![](../images/blog/warn-legacy-context-in-strict-mode.png)
 
 Read the [new context API documentation](/docs/context.html) to help migrate to the new version.
+
+
+### Detecting unsafe effects {#detecting-unsafe-effects}
+
+Conceptually, React effects include three phases:
+* The **create** phase is called for all effects when the effect is created.
+* The **update** phase is called when the dependencies change, if applicable.
+* The **destroy** phase is called for all effects when the effect is destroyed.
```

Comment:
Love this clarification and tying it back to the historical behavior with mounting and unmounting 👍🏻 

Though is there a difference for React developers between update and create?

```suggestion
Conceptually, React effects include three phases:
* The **create** phase is called for all effects when the effect is created i.e. when dependencies change.
* The **destroy** phase is called for all effects when the effect is destroyed.
```
The wording can be polished. It's more about reducing it to two phases: create and destroy.

Reply:
Removed this section